plugin.video.curiositystream/main.py

```diff
@@ -0,0 +1,205 @@
+import functools
```

Comment:
usually this file, as it is the entrypoint should be as small as possible. Just import the main codebase and run. The reason behind it is the fact that this is the only file that's not compiled to bitcode

Reply:
I've reduced as much as possible 👍